yarn.lock

```diff
@@ -17757,6 +17765,13 @@ react-element-to-jsx-string@^14.3.4:
     is-plain-object "5.0.0"
     react-is "17.0.2"
 
+react-error-boundary@^3.1.0:
+  version "3.1.4"
+  resolved "https://registry.yarnpkg.com/react-error-boundary/-/react-error-boundary-3.1.4.tgz#255db92b23197108757a888b01e5b729919abde0"
```

Comment:
interesting that here the link is on `yarnpkg`. when I did yarn upgrade it migrate link to `npmjs.org` from `yarnpkg.com`. maybe we use different version (not an issue)

Reply:
That is strange.